<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget {
	
	background-color:rgb(255,255,255);

}</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>491</width>
      <height>452</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>150</horstretch>
      <verstretch>150</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>150</height>
     </size>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_11">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>150</horstretch>
         <verstretch>150</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>150</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
   border-right: 2px solid black;
   border-bottom: 2px solid black;
   border-top:0px;
   font-size:150px;
   text-align:center;
   padding-bottom:10px;
   padding-right:5px;
   font-family:Comic Sans MS;
 }</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pushButton_23">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>150</horstretch>
         <verstretch>150</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>150</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
   border-bottom: 2px solid black;
   border-top:0px;
   font-size:150px;
   text-align:center;
   padding-bottom:10px;
   padding-right:5px;
   font-family:Comic Sans MS;
 }</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="pushButton_33">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>150</horstretch>
         <verstretch>150</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>150</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
   border-top:0px;
   font-size:150px;
   text-align:center;
   padding-bottom:10px;
   padding-right:5px;
   font-family:Comic Sans MS;
 }</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton_22">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>150</horstretch>
         <verstretch>150</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>150</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
   border-right: 2px solid black;
   border-bottom: 2px solid black;
   border-top:0px;
   font-size:150px;
   text-align:center;
   padding-bottom:10px;
   padding-right:5px;
   font-family:Comic Sans MS;
 }</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_21">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>150</horstretch>
         <verstretch>150</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>150</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
   border-right: 2px solid black;
   border-bottom: 2px solid black;
   border-top:0px;
   font-size:150px;
   text-align:center;
   padding-bottom:10px;
   padding-right:5px;
   font-family:Comic Sans MS;
 }</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton_32">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>150</horstretch>
         <verstretch>150</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>150</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
   border-right: 2px solid black;
   border-top:0px;
   font-size:150px;
   text-align:center;
   padding-bottom:10px;
   padding-right:5px;
   font-family:Comic Sans MS;
 }</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_31">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>150</horstretch>
         <verstretch>150</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>150</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
   border-right: 2px solid black;
   border-top:0px;
   font-size:150px;
   text-align:center;
   padding-bottom:10px;
   padding-right:5px;
   font-family:Comic Sans MS;
 }</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButton_13">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>150</horstretch>
         <verstretch>150</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>150</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
   border-bottom: 2px solid black;
   border-top:0px;
   font-size:150px;
   text-align:center;
   padding-bottom:10px;
   padding-right:5px;
   font-family:Comic Sans MS;
 }</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_12">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>150</horstretch>
         <verstretch>150</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>150</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
   border-right: 2px solid black;
   border-bottom: 2px solid black;
   border-top:0px;
   font-size:150px;
   text-align:center;
   padding-bottom:10px;
   padding-right:5px;
   font-family:Comic Sans MS;
 }</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="turnLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	font-size:30px;
    font-family: Comic Sans MS;

}</string>
    </property>
    <property name="text">
     <string>X TURN</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>540</y>
      <width>421</width>
      <height>92</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="playAgainButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>30</horstretch>
         <verstretch>100</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {

   font-size:25px;
   text-align:center;
   padding-bottom:10px;
   padding-right:5px;
   font-family:Comic Sans MS;
   border:no-border;
   padding-top:5px;
}

QPushButton:hover {
	border-width:2px;
    border-color:black;
	border-style:groove;
    border-radius:15%;
}</string>
       </property>
       <property name="text">
        <string>Play Again</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="goBackButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>30</horstretch>
         <verstretch>100</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {

   font-size:25px;
   text-align:center;
   padding-bottom:5px;
   padding-right:5px;
   font-family:Comic Sans MS;
   border:no-border;
   opacity:0;
   background-color:transparent;
}

QPushButton:hover {
	border-width:2px;
    border-color:black;
	border-style:groove;
    border-radius:15%;
}</string>
       </property>
       <property name="text">
        <string>Go Back</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
